gappy/gap_includes.pxd

```diff
@@ -129,20 +129,29 @@ cdef extern from "gap/gasman.h" nogil:
     """
     #define GAP_CollectBags(full) CollectBags(0, full)
     """
-    void GAP_MarkBag "MarkBag" (Obj bag)
+    void GAP_MarkBag(Obj bag)
     UInt GAP_CollectBags(UInt full)
 
 
 cdef extern from "gap/io.h" nogil:
-    UInt OpenOutputStream(Obj stream)
-    UInt CloseOutput()
+    """
+    TypOutputFile output = {0};
+    inline UInt GAP_OpenOutputStream(Obj stream) {
+        return OpenOutputStream(&output, stream);
+    }
+    inline UInt GAP_CloseOutput( ) {
+        return CloseOutput(&output);
+    }
+    """
+    UInt GAP_OpenOutputStream(Obj stream)
```

Comment:
Why is this a string? Is this a Cython trick I forgot?

Reply:
sorry which string?